cms/templates/js/course-outline.underscore

```diff
@@ -84,8 +84,11 @@ if (xblockInfo.get('graded')) {
 
 var is_proctored_exam = xblockInfo.get('is_proctored_exam');
 var is_practice_exam = xblockInfo.get('is_practice_exam');
+var is_onboarding_exam = xblockInfo.get('is_onboarding_exam');
 if (is_proctored_exam) {
-    if (is_practice_exam) {
+    if (is_onboarding_exam) {
+        var exam_value = gettext('Onboarding Exam');
```

Comment:
Let's be consistent with the eslint rule and hoist this `var` up to be alongside the other declarations in this code block, rather than relying on JS to do stuff other languages wouldn't be chill doing.